outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "rg_name" {
```

Comment:
same regarding prefix ''rg_" here, seems no special need to have it

Reply:
done